deaf-and-hard-of-hearing/event-organizers/conference-best-practices.md

```diff
@@ -0,0 +1,151 @@
+# Best practices for an inclusive and welcoming conference
+
+_This is a living document, so please be sure to check regularly for updates._
+
+If you're not familiar with the needs and preferences of deaf and hard of hearing (deaf/hoh) individuals, hosting a conference that is truly inclusive and accessible can feel overwhelming. This document will guide you to ensuring that deaf and hard of hearing attendees can fully participate in your event and have a great overall and complete experience.  
+
+The first step of being more inclusive is understanding that deaf/hoh are not a one-size-fits-all group — we have different needs and preferences, so it is important to provide multiple options for deaf/hoh. If we can't participate, we will likely not return, reinforcing the vicious cycle of deaf/hoh individuals not participating in the community.  
+
+We also recommend providing attendees, speakers, and sponsors the opportunity to specify their accessibility needs as soon as possible, so you can plan for it in advance. Use your website, registration and call for papers forms to get that information early. Conference organizers often don't budget for accessibility until a request is made. This can create uncomfortable situations for event planners who don't know how to budget for a sudden request. Accessibility is rarely a static budget line item, and it can be hard, especially when planning it for the first time. One of our aims in documenting best practices is to learn and share cost information to make planning more predictable.
+
+To make the process as straightforward as possible, we've listed as much as we could to make a conference truly inclusive to our community but also included alternatives if a particular accommodation is not feasible. If you have any questions or concerns, please get in touch with us at cncf-tag-contributor-strategy@lists.cncf.io. 
+
+In this document, we'll cover:
+
+- [DEIA: Diversity + Equity + Inclusion + Accessibility](#deia-diversity-equity-inclusion-accessibility)
```

Comment:
@JonZeolla, can you help me here? Why aren't these links working? Looks like the same thing you did?

Reply:
No problem
```suggestion
- [DEIA: Diversity + Equity + Inclusion + Accessibility](#deia-diversity--equity--inclusion--accessibility)
```